packages/exchange-connector/src/exchangeDownloader.ts

```diff
@@ -54,7 +57,7 @@ export function downloadRestApis(
   const promises: Promise<any>[] = [];
 
   restApi.forEach((api: RestApi) => {
-    promises.push(downloadRestApi(api, destinationFolder));
+    promises.push(exports.downloadRestApi(api, destinationFolder));
```

Comment:
to help with stubbing

Reply:
I have to say...I don't love this but will accept it if we don't have a better way.

Reply:
I know it's not ideal but it's either this or moving the helper functions to a separate file.

Reply:
I don't think we want this approach, I documented the details in the renderer tests
```javascript
/**
 * Tests all the functions that are invoked while rendering templates.
 *
 * Note: Ideally we want a independent test for each function. As of today (01/30/2020) this is not possible with Sinon when the functions are in the same module.
 * Sinon works by overwriting a reference to a function on an object, making it point at something else. It can't replace references to the same function in other contexts.
 *
 * Example: We can't mock renderApiFamily function while testing renderTemplates function. To make it work renderTemplates should invoke
 * renderApiFamily as "module.exports.renderApiFamily(..)" which is not desired.
 */
```